[2024-12-24T09:37:08.088+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-24T09:37:08.102+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_Calendar_NTBK.NB_Notebook_job manual__2024-12-24T09:37:05.727677+00:00 [queued]>
[2024-12-24T09:37:08.111+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_Calendar_NTBK.NB_Notebook_job manual__2024-12-24T09:37:05.727677+00:00 [queued]>
[2024-12-24T09:37:08.112+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-24T09:37:08.119+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): NB_Notebook_job> on 2024-12-24 09:37:05.727677+00:00
[2024-12-24T09:37:08.125+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=290) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-24T09:37:08.130+0000] {standard_task_runner.py:72} INFO - Started process 293 to run task
[2024-12-24T09:37:08.130+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_Calendar_NTBK', 'NB_Notebook_job', 'manual__2024-12-24T09:37:05.727677+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/dag_calendar.py', '--cfg-path', '/tmp/tmp7gshdanq']
[2024-12-24T09:37:08.131+0000] {standard_task_runner.py:105} INFO - Job 401: Subtask NB_Notebook_job
[2024-12-24T09:37:08.195+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_Calendar_NTBK.NB_Notebook_job manual__2024-12-24T09:37:05.727677+00:00 [running]> on host ca7403376fcd
[2024-12-24T09:37:08.288+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_Calendar_NTBK' AIRFLOW_CTX_TASK_ID='NB_Notebook_job' AIRFLOW_CTX_EXECUTION_DATE='2024-12-24T09:37:05.727677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-24T09:37:05.727677+00:00'
[2024-12-24T09:37:08.290+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-24T09:37:08.886+0000] {docker.py:379} INFO - Starting docker container from image spark-cluster:version-1.0.0
[2024-12-24T09:37:11.307+0000] {docker.py:73} INFO - Input Notebook:  /home/jovyan/Notebooks/NB_Calendar.ipynb
[2024-12-24T09:37:11.312+0000] {docker.py:73} INFO - Output Notebook: /home/jovyan/Notebooks/output/NB_Calendar.ipynb
[2024-12-24T09:37:12.705+0000] {docker.py:73} INFO - Executing notebook with kernel: python3
[2024-12-24T09:37:12.708+0000] {docker.py:73} INFO - Executing Cell 1---------------------------------------
[2024-12-24T09:37:13.204+0000] {docker.py:73} INFO - Ending Cell 1------------------------------------------
[2024-12-24T09:37:13.208+0000] {docker.py:73} INFO - Executing Cell 2---------------------------------------
[2024-12-24T09:37:13.228+0000] {docker.py:73} INFO - Ending Cell 2------------------------------------------
[2024-12-24T09:37:13.230+0000] {docker.py:73} INFO - Executing Cell 3---------------------------------------
[2024-12-24T09:37:15.197+0000] {docker.py:73} INFO - :: loading settings :: url = jar:file:/usr/local/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-12-24T09:37:15.293+0000] {docker.py:73} INFO - Ivy Default Cache set to: /home/jovyan/.ivy2/cache
[2024-12-24T09:37:15.296+0000] {docker.py:73} INFO - The jars for the packages stored in: /home/jovyan/.ivy2/jars
[2024-12-24T09:37:15.300+0000] {docker.py:73} INFO - io.delta#delta-spark_2.12 added as a dependency
[2024-12-24T09:37:15.304+0000] {docker.py:73} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-ef1c390a-750f-46da-9c92-2fd1ecff0f33;1.0
[2024-12-24T09:37:15.308+0000] {docker.py:73} INFO - 	confs: [default]
[2024-12-24T09:37:25.514+0000] {docker.py:73} INFO - 	found io.delta#delta-spark_2.12;3.1.0 in central
[2024-12-24T09:37:25.533+0000] {docker.py:73} INFO - downloading https://repo1.maven.org/maven2/io/delta/delta-spark_2.12/3.1.0/delta-spark_2.12-3.1.0.jar ...
[2024-12-24T09:38:38.191+0000] {docker.py:73} INFO - 	[SUCCESSFUL ] io.delta#delta-spark_2.12;3.1.0!delta-spark_2.12.jar (72652ms)
[2024-12-24T09:38:38.198+0000] {docker.py:73} INFO - :: resolution report :: resolve 10233ms :: artifacts dl 72661ms
[2024-12-24T09:38:38.199+0000] {docker.py:73} INFO - 	:: modules in use:
[2024-12-24T09:38:38.201+0000] {docker.py:73} INFO - 	io.delta#delta-spark_2.12;3.1.0 from central in [default]
[2024-12-24T09:38:38.202+0000] {docker.py:73} INFO - 	---------------------------------------------------------------------
[2024-12-24T09:38:38.204+0000] {docker.py:73} INFO - 	|                  |            modules            ||   artifacts   |
[2024-12-24T09:38:38.205+0000] {docker.py:73} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2024-12-24T09:38:38.207+0000] {docker.py:73} INFO - 	---------------------------------------------------------------------
[2024-12-24T09:38:38.208+0000] {docker.py:73} INFO - 	|      default     |   1   |   1   |   0   |   0   ||   1   |   1   |
[2024-12-24T09:38:38.209+0000] {docker.py:73} INFO - 	---------------------------------------------------------------------
[2024-12-24T09:38:38.211+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:38.213+0000] {docker.py:73} INFO - :: problems summary ::
[2024-12-24T09:38:38.214+0000] {docker.py:73} INFO - :::: ERRORS
[2024-12-24T09:38:38.215+0000] {docker.py:73} INFO - 	Server access error at url https://repo1.maven.org/maven2/io/delta/delta-spark_2.12/3.1.0/delta-spark_2.12-3.1.0.pom (java.net.ConnectException: Connection refused)
[2024-12-24T09:38:38.216+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:38.217+0000] {docker.py:73} INFO - :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
[2024-12-24T09:38:38.218+0000] {docker.py:73} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-ef1c390a-750f-46da-9c92-2fd1ecff0f33
[2024-12-24T09:38:38.219+0000] {docker.py:73} INFO - 	confs: [default]
[2024-12-24T09:38:38.245+0000] {docker.py:73} INFO - 	1 artifacts copied, 0 already retrieved (5374kB/31ms)
[2024-12-24T09:38:38.460+0000] {docker.py:73} INFO - 24/12/24 09:38:38 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-12-24T09:38:38.611+0000] {docker.py:73} INFO - Setting default log level to "WARN".
[2024-12-24T09:38:38.612+0000] {docker.py:73} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2024-12-24T09:38:39.158+0000] {docker.py:73} INFO - 24/12/24 09:38:39 WARN StandaloneSchedulerBackend: Dynamic allocation enabled without spark.executor.cores explicitly set, you may get more executors allocated than expected. It's recommended to set spark.executor.cores explicitly. Please check SPARK-30299 for more details.
[2024-12-24T09:38:39.481+0000] {docker.py:73} INFO - 24/12/24 09:38:39 WARN Utils: spark.executor.instances less than spark.dynamicAllocation.minExecutors is invalid, ignoring its setting, please update your configs.
[2024-12-24T09:38:40.082+0000] {docker.py:73} INFO - Driver Python Version: sys.version_info(major=3, minor=11, micro=6, releaselevel='final', serial=0)
[2024-12-24T09:38:40.084+0000] {docker.py:73} INFO - Worker Python Version: /opt/conda/bin/python
[2024-12-24T09:38:40.417+0000] {docker.py:73} INFO - <pyspark.sql.session.SparkSession at 0xffff649287d0>
[2024-12-24T09:38:40.421+0000] {docker.py:73} INFO - Ending Cell 3------------------------------------------
[2024-12-24T09:38:40.425+0000] {docker.py:73} INFO - Executing Cell 4---------------------------------------
[2024-12-24T09:38:40.437+0000] {docker.py:73} INFO - Ending Cell 4------------------------------------------
[2024-12-24T09:38:40.440+0000] {docker.py:73} INFO - Executing Cell 5---------------------------------------
[2024-12-24T09:38:40.448+0000] {docker.py:73} INFO - Ending Cell 5------------------------------------------
[2024-12-24T09:38:40.452+0000] {docker.py:73} INFO - Executing Cell 6---------------------------------------
[2024-12-24T09:38:41.261+0000] {docker.py:73} INFO - Ending Cell 6------------------------------------------
[2024-12-24T09:38:41.264+0000] {docker.py:73} INFO - Executing Cell 7---------------------------------------
[2024-12-24T09:38:41.448+0000] {docker.py:73} INFO - Ending Cell 7------------------------------------------
[2024-12-24T09:38:41.459+0000] {docker.py:73} INFO - Executing Cell 8---------------------------------------
[2024-12-24T09:38:41.592+0000] {docker.py:73} INFO - Ending Cell 8------------------------------------------
[2024-12-24T09:38:41.595+0000] {docker.py:73} INFO - Executing Cell 9---------------------------------------
[2024-12-24T09:38:43.113+0000] {docker.py:73} INFO - [Stage 0:>                                                          (0 + 2) / 2]
[2024-12-24T09:38:44.886+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:44.929+0000] {docker.py:73} INFO - 18628
[2024-12-24T09:38:44.930+0000] {docker.py:73} INFO - Ending Cell 9------------------------------------------
[2024-12-24T09:38:44.934+0000] {docker.py:73} INFO - Executing Cell 10--------------------------------------
[2024-12-24T09:38:45.024+0000] {docker.py:73} INFO - Ending Cell 10-----------------------------------------
[2024-12-24T09:38:45.025+0000] {docker.py:73} INFO - Executing Cell 11--------------------------------------
[2024-12-24T09:38:45.530+0000] {docker.py:73} INFO - Ending Cell 11-----------------------------------------
[2024-12-24T09:38:45.692+0000] {docker.py:73} INFO - 24/12/24 09:38:45 WARN JavaUtils: Attempt to delete using native Unix OS command failed for path = /tmp/spark-eff1e57f-1d40-4fca-8942-9bc3eb990ac6. Falling back to Java IO way
[2024-12-24T09:38:45.694+0000] {docker.py:73} INFO - java.io.IOException: Failed to delete: /tmp/spark-eff1e57f-1d40-4fca-8942-9bc3eb990ac6
[2024-12-24T09:38:45.695+0000] {docker.py:73} INFO - 	at org.apache.spark.network.util.JavaUtils.deleteRecursivelyUsingUnixNative(JavaUtils.java:173)
[2024-12-24T09:38:45.695+0000] {docker.py:73} INFO - 	at org.apache.spark.network.util.JavaUtils.deleteRecursively(JavaUtils.java:109)
[2024-12-24T09:38:45.696+0000] {docker.py:73} INFO - 	at org.apache.spark.network.util.JavaUtils.deleteRecursively(JavaUtils.java:90)
[2024-12-24T09:38:45.697+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkFileUtils.deleteRecursively(SparkFileUtils.scala:121)
[2024-12-24T09:38:45.698+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkFileUtils.deleteRecursively$(SparkFileUtils.scala:120)
[2024-12-24T09:38:45.699+0000] {docker.py:73} INFO - 	at org.apache.spark.util.Utils$.deleteRecursively(Utils.scala:1126)
[2024-12-24T09:38:45.700+0000] {docker.py:73} INFO - 	at org.apache.spark.util.ShutdownHookManager$.$anonfun$new$4(ShutdownHookManager.scala:65)
[2024-12-24T09:38:45.700+0000] {docker.py:73} INFO - 	at org.apache.spark.util.ShutdownHookManager$.$anonfun$new$4$adapted(ShutdownHookManager.scala:62)
[2024-12-24T09:38:45.701+0000] {docker.py:73} INFO - 	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:36)
[2024-12-24T09:38:45.703+0000] {docker.py:73} INFO - 	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:33)
[2024-12-24T09:38:45.704+0000] {docker.py:73} INFO - 	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:198)
[2024-12-24T09:38:45.705+0000] {docker.py:73} INFO - 	at org.apache.spark.util.ShutdownHookManager$.$anonfun$new$2(ShutdownHookManager.scala:62)
[2024-12-24T09:38:45.705+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkShutdownHook.run(ShutdownHookManager.scala:214)
[2024-12-24T09:38:45.706+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkShutdownHookManager.$anonfun$runAll$2(ShutdownHookManager.scala:188)
[2024-12-24T09:38:45.708+0000] {docker.py:73} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-12-24T09:38:45.709+0000] {docker.py:73} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:1928)
[2024-12-24T09:38:45.712+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkShutdownHookManager.$anonfun$runAll$1(ShutdownHookManager.scala:188)
[2024-12-24T09:38:45.714+0000] {docker.py:73} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-12-24T09:38:45.714+0000] {docker.py:73} INFO - 	at scala.util.Try$.apply(Try.scala:213)
[2024-12-24T09:38:45.715+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkShutdownHookManager.runAll(ShutdownHookManager.scala:188)
[2024-12-24T09:38:45.716+0000] {docker.py:73} INFO - 	at org.apache.spark.util.SparkShutdownHookManager$$anon$2.run(ShutdownHookManager.scala:178)
[2024-12-24T09:38:45.717+0000] {docker.py:73} INFO - 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2024-12-24T09:38:45.719+0000] {docker.py:73} INFO - 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2024-12-24T09:38:45.719+0000] {docker.py:73} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-24T09:38:45.721+0000] {docker.py:73} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-24T09:38:45.721+0000] {docker.py:73} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-24T09:38:48.207+0000] {docker.py:73} INFO - Traceback (most recent call last):
[2024-12-24T09:38:48.217+0000] {docker.py:73} INFO -   File "/opt/conda/bin/papermill", line 8, in <module>
[2024-12-24T09:38:48.218+0000] {docker.py:73} INFO -     sys.exit(papermill())
[2024-12-24T09:38:48.219+0000] {docker.py:73} INFO -              ^^^^^^^^^^^
[2024-12-24T09:38:48.220+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
[2024-12-24T09:38:48.221+0000] {docker.py:73} INFO -     return self.main(*args, **kwargs)
[2024-12-24T09:38:48.221+0000] {docker.py:73} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-12-24T09:38:48.222+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/click/core.py", line 1078, in main
[2024-12-24T09:38:48.222+0000] {docker.py:73} INFO -     rv = self.invoke(ctx)
[2024-12-24T09:38:48.223+0000] {docker.py:73} INFO -          ^^^^^^^^^^^^^^^^
[2024-12-24T09:38:48.223+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
[2024-12-24T09:38:48.223+0000] {docker.py:73} INFO -     return ctx.invoke(self.callback, **ctx.params)
[2024-12-24T09:38:48.224+0000] {docker.py:73} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-12-24T09:38:48.224+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/click/core.py", line 783, in invoke
[2024-12-24T09:38:48.224+0000] {docker.py:73} INFO -     return __callback(*args, **kwargs)
[2024-12-24T09:38:48.224+0000] {docker.py:73} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-12-24T09:38:48.225+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/click/decorators.py", line 33, in new_func
[2024-12-24T09:38:48.225+0000] {docker.py:73} INFO -     return f(get_current_context(), *args, **kwargs)
[2024-12-24T09:38:48.225+0000] {docker.py:73} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-12-24T09:38:48.225+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/papermill/cli.py", line 235, in papermill
[2024-12-24T09:38:48.225+0000] {docker.py:73} INFO -     execute_notebook(
[2024-12-24T09:38:48.226+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/papermill/execute.py", line 131, in execute_notebook
[2024-12-24T09:38:48.226+0000] {docker.py:73} INFO -     raise_for_execution_errors(nb, output_path)
[2024-12-24T09:38:48.226+0000] {docker.py:73} INFO -   File "/opt/conda/lib/python3.11/site-packages/papermill/execute.py", line 251, in raise_for_execution_errors
[2024-12-24T09:38:48.226+0000] {docker.py:73} INFO -     raise error
[2024-12-24T09:38:48.227+0000] {docker.py:73} INFO - papermill.exceptions.PapermillExecutionError: 
[2024-12-24T09:38:48.227+0000] {docker.py:73} INFO - ---------------------------------------------------------------------------
[2024-12-24T09:38:48.228+0000] {docker.py:73} INFO - Exception encountered at "In [11]":
[2024-12-24T09:38:48.229+0000] {docker.py:73} INFO - ---------------------------------------------------------------------------
[2024-12-24T09:38:48.229+0000] {docker.py:73} INFO - Py4JJavaError                             Traceback (most recent call last)
[2024-12-24T09:38:48.230+0000] {docker.py:73} INFO - Cell In[11], line 1
[2024-12-24T09:38:48.231+0000] {docker.py:73} INFO - ----> 1 if DeltaTable.isDeltaTable(spark, trgt_path_processed):
[2024-12-24T09:38:48.232+0000] {docker.py:73} INFO -       2     column_name = df_output.columns
[2024-12-24T09:38:48.233+0000] {docker.py:73} INFO -       3     set_clause = ", ".join([f"target.{i} = source.{i}" for i in column_name])
[2024-12-24T09:38:48.234+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.235+0000] {docker.py:73} INFO - File /opt/conda/lib/python3.11/site-packages/delta/tables.py:562, in DeltaTable.isDeltaTable(cls, sparkSession, identifier)
[2024-12-24T09:38:48.235+0000] {docker.py:73} INFO -     559 jvm: "JVMView" = sparkSession._sc._jvm  # type: ignore[attr-defined]
[2024-12-24T09:38:48.235+0000] {docker.py:73} INFO -     560 jsparkSession: "JavaObject" = sparkSession._jsparkSession  # type: ignore[attr-defined]
[2024-12-24T09:38:48.236+0000] {docker.py:73} INFO - --> 562 return jvm.io.delta.tables.DeltaTable.isDeltaTable(jsparkSession, identifier)
[2024-12-24T09:38:48.237+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.237+0000] {docker.py:73} INFO - File /opt/conda/lib/python3.11/site-packages/py4j/java_gateway.py:1322, in JavaMember.__call__(self, *args)
[2024-12-24T09:38:48.238+0000] {docker.py:73} INFO -    1316 command = proto.CALL_COMMAND_NAME +\
[2024-12-24T09:38:48.238+0000] {docker.py:73} INFO -    1317     self.command_header +\
[2024-12-24T09:38:48.238+0000] {docker.py:73} INFO -    1318     args_command +\
[2024-12-24T09:38:48.238+0000] {docker.py:73} INFO -    1319     proto.END_COMMAND_PART
[2024-12-24T09:38:48.239+0000] {docker.py:73} INFO -    1321 answer = self.gateway_client.send_command(command)
[2024-12-24T09:38:48.240+0000] {docker.py:73} INFO - -> 1322 return_value = get_return_value(
[2024-12-24T09:38:48.240+0000] {docker.py:73} INFO -    1323     answer, self.gateway_client, self.target_id, self.name)
[2024-12-24T09:38:48.241+0000] {docker.py:73} INFO -    1325 for temp_arg in temp_args:
[2024-12-24T09:38:48.241+0000] {docker.py:73} INFO -    1326     if hasattr(temp_arg, "_detach"):
[2024-12-24T09:38:48.242+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.243+0000] {docker.py:73} INFO - File /opt/conda/lib/python3.11/site-packages/pyspark/errors/exceptions/captured.py:179, in capture_sql_exception.<locals>.deco(*a, **kw)
[2024-12-24T09:38:48.244+0000] {docker.py:73} INFO -     177 def deco(*a: Any, **kw: Any) -> Any:
[2024-12-24T09:38:48.244+0000] {docker.py:73} INFO -     178     try:
[2024-12-24T09:38:48.244+0000] {docker.py:73} INFO - --> 179         return f(*a, **kw)
[2024-12-24T09:38:48.245+0000] {docker.py:73} INFO -     180     except Py4JJavaError as e:
[2024-12-24T09:38:48.245+0000] {docker.py:73} INFO -     181         converted = convert_exception(e.java_exception)
[2024-12-24T09:38:48.246+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.246+0000] {docker.py:73} INFO - File /opt/conda/lib/python3.11/site-packages/py4j/protocol.py:326, in get_return_value(answer, gateway_client, target_id, name)
[2024-12-24T09:38:48.247+0000] {docker.py:73} INFO -     324 value = OUTPUT_CONVERTER[type](answer[2:], gateway_client)
[2024-12-24T09:38:48.248+0000] {docker.py:73} INFO -     325 if answer[1] == REFERENCE_TYPE:
[2024-12-24T09:38:48.248+0000] {docker.py:73} INFO - --> 326     raise Py4JJavaError(
[2024-12-24T09:38:48.248+0000] {docker.py:73} INFO -     327         "An error occurred while calling {0}{1}{2}.\n".
[2024-12-24T09:38:48.249+0000] {docker.py:73} INFO -     328         format(target_id, ".", name), value)
[2024-12-24T09:38:48.249+0000] {docker.py:73} INFO -     329 else:
[2024-12-24T09:38:48.250+0000] {docker.py:73} INFO -     330     raise Py4JError(
[2024-12-24T09:38:48.251+0000] {docker.py:73} INFO -     331         "An error occurred while calling {0}{1}{2}. Trace:\n{3}\n".
[2024-12-24T09:38:48.252+0000] {docker.py:73} INFO -     332         format(target_id, ".", name, value))
[2024-12-24T09:38:48.253+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.254+0000] {docker.py:73} INFO - Py4JJavaError: An error occurred while calling z:io.delta.tables.DeltaTable.isDeltaTable.
[2024-12-24T09:38:48.255+0000] {docker.py:73} INFO - : com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: io/delta/storage/LogStore
[2024-12-24T09:38:48.255+0000] {docker.py:73} INFO - Please ensure that the delta-storage dependency is included.
[2024-12-24T09:38:48.256+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.257+0000] {docker.py:73} INFO - If using Python, please ensure you call `configure_spark_with_delta_pip` or use
[2024-12-24T09:38:48.257+0000] {docker.py:73} INFO - `--packages io.delta:delta-spark_<scala-version>:<delta-lake-version>`.
[2024-12-24T09:38:48.258+0000] {docker.py:73} INFO - See https://docs.delta.io/latest/quick-start.html#python.
[2024-12-24T09:38:48.258+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.259+0000] {docker.py:73} INFO - More information about this dependency and how to include it can be found here:
[2024-12-24T09:38:48.259+0000] {docker.py:73} INFO - https://docs.delta.io/latest/porting.html#delta-lake-1-1-or-below-to-delta-lake-1-2-or-above.
[2024-12-24T09:38:48.260+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.260+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2261)
[2024-12-24T09:38:48.261+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache.get(LocalCache.java:4000)
[2024-12-24T09:38:48.261+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4789)
[2024-12-24T09:38:48.261+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.getDeltaLogFromCache$1(DeltaLog.scala:790)
[2024-12-24T09:38:48.262+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.apply(DeltaLog.scala:800)
[2024-12-24T09:38:48.262+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.apply(DeltaLog.scala:710)
[2024-12-24T09:38:48.263+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.forTable(DeltaLog.scala:672)
[2024-12-24T09:38:48.263+0000] {docker.py:73} INFO - 	at io.delta.tables.DeltaTable$.isDeltaTable(DeltaTable.scala:811)
[2024-12-24T09:38:48.264+0000] {docker.py:73} INFO - 	at io.delta.tables.DeltaTable.isDeltaTable(DeltaTable.scala)
[2024-12-24T09:38:48.264+0000] {docker.py:73} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-12-24T09:38:48.265+0000] {docker.py:73} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-12-24T09:38:48.265+0000] {docker.py:73} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-12-24T09:38:48.266+0000] {docker.py:73} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-12-24T09:38:48.266+0000] {docker.py:73} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-12-24T09:38:48.267+0000] {docker.py:73} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-12-24T09:38:48.267+0000] {docker.py:73} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2024-12-24T09:38:48.267+0000] {docker.py:73} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-12-24T09:38:48.268+0000] {docker.py:73} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-12-24T09:38:48.269+0000] {docker.py:73} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-12-24T09:38:48.270+0000] {docker.py:73} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-12-24T09:38:48.271+0000] {docker.py:73} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-24T09:38:48.272+0000] {docker.py:73} INFO - Caused by: java.lang.NoClassDefFoundError: io/delta/storage/LogStore
[2024-12-24T09:38:48.272+0000] {docker.py:73} INFO - Please ensure that the delta-storage dependency is included.
[2024-12-24T09:38:48.273+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.274+0000] {docker.py:73} INFO - If using Python, please ensure you call `configure_spark_with_delta_pip` or use
[2024-12-24T09:38:48.274+0000] {docker.py:73} INFO - `--packages io.delta:delta-spark_<scala-version>:<delta-lake-version>`.
[2024-12-24T09:38:48.274+0000] {docker.py:73} INFO - See https://docs.delta.io/latest/quick-start.html#python.
[2024-12-24T09:38:48.275+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.275+0000] {docker.py:73} INFO - More information about this dependency and how to include it can be found here:
[2024-12-24T09:38:48.275+0000] {docker.py:73} INFO - https://docs.delta.io/latest/porting.html#delta-lake-1-1-or-below-to-delta-lake-1-2-or-above.
[2024-12-24T09:38:48.276+0000] {docker.py:73} INFO - 
[2024-12-24T09:38:48.277+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaErrorsBase.missingDeltaStorageJar(DeltaErrors.scala:2659)
[2024-12-24T09:38:48.278+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaErrorsBase.missingDeltaStorageJar$(DeltaErrors.scala:2656)
[2024-12-24T09:38:48.278+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaErrors$.missingDeltaStorageJar(DeltaErrors.scala:3203)
[2024-12-24T09:38:48.279+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore$.<init>(DelegatingLogStore.scala:163)
[2024-12-24T09:38:48.280+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore$.<clinit>(DelegatingLogStore.scala)
[2024-12-24T09:38:48.280+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore.$anonfun$schemeBasedLogStore$1(DelegatingLogStore.scala:67)
[2024-12-24T09:38:48.281+0000] {docker.py:73} INFO - 	at scala.Option.orElse(Option.scala:447)
[2024-12-24T09:38:48.281+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore.schemeBasedLogStore(DelegatingLogStore.scala:67)
[2024-12-24T09:38:48.282+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore.getDelegate(DelegatingLogStore.scala:85)
[2024-12-24T09:38:48.283+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.storage.DelegatingLogStore.read(DelegatingLogStore.scala:96)
[2024-12-24T09:38:48.284+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.Checkpoints.$anonfun$loadMetadataFromFile$1(Checkpoints.scala:361)
[2024-12-24T09:38:48.285+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile(DeltaLogging.scala:140)
[2024-12-24T09:38:48.285+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile$(DeltaLogging.scala:138)
[2024-12-24T09:38:48.286+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.recordFrameProfile(DeltaLog.scala:72)
[2024-12-24T09:38:48.286+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.$anonfun$recordDeltaOperationInternal$1(DeltaLogging.scala:133)
[2024-12-24T09:38:48.287+0000] {docker.py:73} INFO - 	at com.databricks.spark.util.DatabricksLogging.recordOperation(DatabricksLogging.scala:128)
[2024-12-24T09:38:48.287+0000] {docker.py:73} INFO - 	at com.databricks.spark.util.DatabricksLogging.recordOperation$(DatabricksLogging.scala:117)
[2024-12-24T09:38:48.288+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.recordOperation(DeltaLog.scala:72)
[2024-12-24T09:38:48.288+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperationInternal(DeltaLogging.scala:132)
[2024-12-24T09:38:48.289+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperation(DeltaLogging.scala:122)
[2024-12-24T09:38:48.290+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperation$(DeltaLogging.scala:112)
[2024-12-24T09:38:48.291+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.recordDeltaOperation(DeltaLog.scala:72)
[2024-12-24T09:38:48.291+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.Checkpoints.loadMetadataFromFile(Checkpoints.scala:360)
[2024-12-24T09:38:48.292+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.Checkpoints.readLastCheckpointFile(Checkpoints.scala:354)
[2024-12-24T09:38:48.293+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.Checkpoints.readLastCheckpointFile$(Checkpoints.scala:353)
[2024-12-24T09:38:48.294+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.readLastCheckpointFile(DeltaLog.scala:72)
[2024-12-24T09:38:48.295+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.SnapshotManagement.$anonfun$getSnapshotAtInit$1(SnapshotManagement.scala:436)
[2024-12-24T09:38:48.296+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile(DeltaLogging.scala:140)
[2024-12-24T09:38:48.300+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile$(DeltaLogging.scala:138)
[2024-12-24T09:38:48.301+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.recordFrameProfile(DeltaLog.scala:72)
[2024-12-24T09:38:48.301+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.SnapshotManagement.getSnapshotAtInit(SnapshotManagement.scala:434)
[2024-12-24T09:38:48.302+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.SnapshotManagement.getSnapshotAtInit$(SnapshotManagement.scala:433)
[2024-12-24T09:38:48.303+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.getSnapshotAtInit(DeltaLog.scala:72)
[2024-12-24T09:38:48.303+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.SnapshotManagement.$init$(SnapshotManagement.scala:60)
[2024-12-24T09:38:48.304+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog.<init>(DeltaLog.scala:78)
[2024-12-24T09:38:48.304+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.$anonfun$apply$4(DeltaLog.scala:779)
[2024-12-24T09:38:48.304+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper$.allowInvokingTransformsInAnalyzer(AnalysisHelper.scala:323)
[2024-12-24T09:38:48.304+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.$anonfun$apply$3(DeltaLog.scala:774)
[2024-12-24T09:38:48.305+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile(DeltaLogging.scala:140)
[2024-12-24T09:38:48.305+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordFrameProfile$(DeltaLogging.scala:138)
[2024-12-24T09:38:48.305+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.recordFrameProfile(DeltaLog.scala:598)
[2024-12-24T09:38:48.306+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.$anonfun$recordDeltaOperationInternal$1(DeltaLogging.scala:133)
[2024-12-24T09:38:48.306+0000] {docker.py:73} INFO - 	at com.databricks.spark.util.DatabricksLogging.recordOperation(DatabricksLogging.scala:128)
[2024-12-24T09:38:48.307+0000] {docker.py:73} INFO - 	at com.databricks.spark.util.DatabricksLogging.recordOperation$(DatabricksLogging.scala:117)
[2024-12-24T09:38:48.307+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.recordOperation(DeltaLog.scala:598)
[2024-12-24T09:38:48.308+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperationInternal(DeltaLogging.scala:132)
[2024-12-24T09:38:48.309+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperation(DeltaLogging.scala:122)
[2024-12-24T09:38:48.310+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.metering.DeltaLogging.recordDeltaOperation$(DeltaLogging.scala:112)
[2024-12-24T09:38:48.311+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.recordDeltaOperation(DeltaLog.scala:598)
[2024-12-24T09:38:48.312+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.createDeltaLog$1(DeltaLog.scala:773)
[2024-12-24T09:38:48.313+0000] {docker.py:73} INFO - 	at org.apache.spark.sql.delta.DeltaLog$.$anonfun$apply$5(DeltaLog.scala:791)
[2024-12-24T09:38:48.313+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4792)
[2024-12-24T09:38:48.314+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3599)
[2024-12-24T09:38:48.314+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2379)
[2024-12-24T09:38:48.314+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2342)
[2024-12-24T09:38:48.315+0000] {docker.py:73} INFO - 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2257)
[2024-12-24T09:38:48.315+0000] {docker.py:73} INFO - 	... 20 more
[2024-12-24T09:38:48.688+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 519, in execute
    return self._run_image()
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 396, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 453, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2024-12-24T09:38:48.706+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=dag_Calendar_NTBK, task_id=NB_Notebook_job, run_id=manual__2024-12-24T09:37:05.727677+00:00, execution_date=20241224T093705, start_date=20241224T093708, end_date=20241224T093848
[2024-12-24T09:38:48.769+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-24T09:38:48.770+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 401 for task NB_Notebook_job (Docker container failed: {'StatusCode': 1}; 293)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 519, in execute
    return self._run_image()
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 396, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 453, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2024-12-24T09:38:48.822+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-24T09:38:48.887+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-24T09:38:48.889+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
